client/components/Coverages/CoverageFieldsRow.tsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import {Spacer, Select, IconButton, Option} from 'superdesk-ui-framework/react';
+import {getVocabularyItemFieldTranslated} from '../../utils/vocabularies';
+import {getUserInterfaceLanguageFromCV} from '../../utils/users';
+import {gettext} from '../../utils';
+import {superdeskApi} from '../../superdeskApi';
+import {IPlanningNewsCoverageStatus} from '../../interfaces';
+import {IDesk, IUser} from 'superdesk-api';
+import {ICoverageLineItem} from './CoverageAddAdvancedModal';
+
+interface IProps {
+    index: number;
+    coverage: Partial<ICoverageLineItem>;
+    newsCoverageStatus: Array<IPlanningNewsCoverageStatus>;
+    languages: Array<any>;
+    handleDeskChange: (coverage: Partial<ICoverageLineItem>, desk: IDesk) => void;
+    handleUserChange: (coverage: Partial<ICoverageLineItem>, user: IUser) => void;
+    updateCoverage: (coverage: Partial<ICoverageLineItem>, updates: Partial<ICoverageLineItem>) => void;
+    duplicateCoverage: (index: number, coverage: Partial<ICoverageLineItem>) => void;
+}
+
+export const CoverageEditableFields: React.FC<IProps> = ({
+    index,
+    coverage,
+    newsCoverageStatus,
+    languages,
+    handleDeskChange,
+    handleUserChange,
+    updateCoverage,
+    duplicateCoverage,
+}) => {
+    const language = getUserInterfaceLanguageFromCV();
+    const {SelectUser} = superdeskApi.components;
+
+    return (
+        <>
+            <Spacer
+                h
+                gap="8"
+                noWrap
+                alignItems="end"
+                style={{padding: 'var(--gap-1)'}}
+                justifyContent="space-between"
+            >
+                <Select
+                    fullWidth
+                    label={gettext('Desk')}
+                    value={coverage.desk?._id}
+                    onChange={(newDeskId) => {
+                        handleDeskChange(
+                            coverage,
+                            coverage.filteredDesks.find(({_id}) => _id === newDeskId),
+                        );
+                    }}
+                >
+                    <Option />
```

Comment:
Is this just an empty option? if so, maybe we should add a label to it like: "Select an option"?